.buildkite/filebeat/filebeat-pipeline.yml

```diff
@@ -124,8 +124,10 @@ steps:
 
   - group: "Filebeat Extended Tests"
     key: "filebeat-extended-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*(macOS|arm).*/
+
     steps:
-      - label: ":mac: MacOS x64_64 Unit Tests"
+      - label: ":mac: Filebeat macOS x86_64 Unit Tests"
```

Comment:
```suggestion
      - label: ":mac: Filebeat MacOS x86_64 Unit Tests"
```

Reply:
It was requested/agreed before to have it as `macOS` to match naming convention. cc: @dliappis

Reply:
Yes, the operating system (since 2016 or so) is capitalized as `macOS` https://en.wikipedia.org/wiki/MacOS fyi @pierrehilbert

Reply:
My bad, I wasn't aware, thanks for the bit of history you shared with me!!

Reply:
> My bad, I wasn't aware, thanks for the bit of history you shared with me!!

No probs! It has been changing over the years for sure!